### ISSUE TITLE
Wrong removeEventListener() parameters

### ISSUE THREAD

**Issue by tkrotoff** [microsoft/TypeScript-DOM-lib-generator#174]
[Current definition](https://github.com/Microsoft/TSJS-lib-generator/blob/1592fac1e22b5a40ca53f2edb8d242d71087e79b/baselines/dom.generated.d.ts#L3358):
```TypeScript
addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;
removeEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;
```

Should be something like:
```TypeScript
interface AddEventListenerOptions {
  capture?: boolean;
  once?: boolean;
  passive?: boolean;
}

interface RemoveEventListenerOptions {
  capture?: boolean;
  passive?: boolean;
}

interface EventTarget {
  addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: AddEventListenerOptions): void;
  addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
  removeEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
  removeEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: RemoveEventListenerOptions): void;
}
```

`listener` is mandatory.

See [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) and [removeEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener):
```JavaScript
target.addEventListener(type, listener[, options]);
target.addEventListener(type, listener[, useCapture]);

target.removeEventListener(type, listener[, options]);
target.removeEventListener(type, listener[, useCapture]);
```

Example:
```JavaScript
var listener = function() {
  console.log('Hello!');
}

document.body.addEventListener('click', listener);

document.body.click(); // Hello!

document.body.removeEventListener('click'); // BOOM! =>
/* Under Chrome 55:
Uncaught TypeError: Failed to execute 'removeEventListener' on 'EventTarget': 2 arguments required, but only 1 present.
*/

/* Under Firefox 47
TypeError: Not enough arguments to EventTarget.removeEventListener.
*/

document.body.addEventListener('click'); // BOOM! =>
/* Under Chrome 55:
Uncaught TypeError: Failed to execute 'addEventListener' on 'EventTarget': 2 arguments required, but only 1 present.
*/
```

**Comment by SaschaNaz** [comment]
Partially fixed by #224 but the listener is still optional.

**Comment by saschanaz** [comment]
Has been fixed since then.

@github-actions close
